code/modules/projectiles/gun.dm

```diff
@@ -201,7 +201,7 @@
 
 /obj/item/weapon/gun/proc/process_fire(atom/target as mob|obj|turf, mob/living/user as mob|obj, message = 1, params, zone_override)
 	add_fingerprint(user)
-
+	add_logs(usr, target, "shot at", admin=0)
```

Comment:
um no, why are you using usr instead of user which is in arguments,plus what the hell is that admin parameter